src/network/RTypeClient.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** RTypeClient
+*/
+
+#ifndef RTYPE_CLIENT_HPP_
+    #define RTYPE_CLIENT_HPP_
+    #include <unordered_map>
```

Comment:
Add a space between define and include plz

Reply:
Done